index.php

```diff
@@ -63,6 +64,17 @@
 
 // Current locks.
 echo html_writer::tag('h1', get_string('h1_current', 'tool_lockstats') . ' (' . count($records) . ')');
+$url = new moodle_url("/admin/tool/lockstats/index.php", array('action' => 'releaseall'));
```

Comment:
@brendanheywood  I'm not sure why I don't need 'sesskey' => sesskey() here. require_sesskey() is still returning true.

Reply:
The single_button form does include sesskey for us, so we don't have to include it here